composer.lock

```diff
@@ -2082,16 +2082,16 @@
         },
         {
             "name": "mccool/laravel-auto-presenter",
-            "version": "4.2.1",
+            "version": "v4.2.1",
```

Comment:
I'm telling you, I've removed `vendor`, `composer.lock`, ran `composer clear-cache`. Numerous times.